static/src/js/components/AccessMethod/AccessMethod.js

```diff
@@ -266,7 +293,8 @@ export class AccessMethod extends Component {
           title = 'Customize'
           subtitle = 'Harmony'
           description = 'Select options like variables, transformations, and output formats for in-region cloud access.'
-          details = `The requested data will be processed using the ${name} service and stored in the cloud for analysis.`
+          details = 'The requested data will be processed using the Harmony service and stored in the cloud for analysis.'
```

Comment:
Is "Harmony" the same as `name`? If `name` is coming from the access method I don't think it would be "Harmony"

Reply:
I removed it and made the test a bit more tight to control changes to this string